src/common/components/WalletBalanceInput/WalletSelectorMenu.tsx

```diff
@@ -0,0 +1,77 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Button,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  VStack,
+} from '@chakra-ui/react';
+
+import { Text } from '../Typography';
+
+export interface ITokenBalance {
+  symbol: string;
+  amount: string;
+  icon: React.ReactNode;
+}
+
+interface WalletBalanceInputProps {
```

Comment:
Same as above, could we export the interface of the component props too?

Reply:
why? its not used anywhere else

Reply:
Resolved, the answer [here](https://github.com/keypom/keypom-airfoil/pull/17#discussion_r1059294282)